extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -525,16 +512,21 @@ export class SchemaCompareDialog {
 			this.dialog.okButton.enabled = await this.shouldEnableOkayButton();
 		});
 
-		// if source is currently a db, show it in the server and db dropdowns
-		if (this.schemaCompareMainWindow.sourceEndpointInfo && this.schemaCompareMainWindow.sourceEndpointInfo.endpointType === mssql.SchemaCompareEndpointType.Database) {
+		if (this.schemaCompareMainWindow.sourceEndpointInfo) {
```

Comment:
any reason why you aren't using a switch statement consistently in this PR?

Reply:
Hmm... not particularly.  Lemme massage it a bit.